File: Pratica 6 - Funcoes 2/java_logic/Exercicio9_v1.java
import java.util.Scanner;

public class Exercicio9_v1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite uma frase ou uma palavra: ");
        String texto = sc.nextLine().toLowerCase();
        int cont = 0;

        for (char c : texto.toCharArray()) {
            if ("aeiou".indexOf(c) != -1) cont++;
        }

        System.out.println("Vogais: "+ cont);

        sc.close();
    }
}

File: Pratica 6 - Funcoes 2/cpp_logic/exercicio9_v1.cpp
#include <iostream>
using namespace std;

int main() {
    string texto;
    cout << "Digite um texto: ";
    cin >> texto;

    int cont = 0;

    for (char c : texto) {
        if (c=='a'||c=='e'||c=='i'||c=='o'||c=='u')
            cont++;
    }

    cout <<"Vogais: " << cont;
    return 0;
}